### PR TITLE
Update Gradle to 6.8.2

### PR DESCRIPTION
**Pull request by gongled**
### What's this PR about

I've updated Gradle to 6.8.2 to be able to update Kafka to 2.7.0.

### Known build problems and traps

None.

### TODO's:

- [x] Run [`perfecto`](https://github.com/essentialkaos/perfecto) check on your spec file
- [ ] Write [`bibop`](https://github.com/essentialkaos/bibop) tests
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

**Did you try to build the package with this spec?:** Yes
**Is this ready for review?:** Yes

### CODE REVIEW

File: specs/gradle/SOURCES/checksum.sha512 [hunk 1]
```diff
@@ -0,0 +1 @@
+885961b001fdfe221511f66d81f2962adffa6d3a5fd8148dd20591e8ff2222135216fb67fffa4248c814b4e059097f2b767c7e023b6249fbf36d558bf3674b61  gradle-6.8.2-src.zip
```
Comment (andyone) [inline specs/gradle/SOURCES/checksum.sha512]: `checksum.sha512` is not required for the build process. This file automatically generated by `rpmbuilder` while building packages. Also, this spec doesn't have a macro for checking these checksums.

File: specs/gradle/gradle.spec [hunk 2]
```diff
@@ -61,9 +61,9 @@ Source100:            checksum.sha512
 
 BuildRoot:            %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:        jdk11 git
+BuildRequires:        jdk13 git
```
Comment (andyone) [inline specs/gradle/gradle.spec:64]: We don't longer provide non-LTS packages with Java. Currently, the repository contains only JDK8 and JDK11 packages.

**Review by andyone — CHANGES_REQUESTED** [review]

**Comment by andyone** [comment]
@gongled Hi! I hope you doing well. Is there any news about this PR?

**Comment by gongled** [comment]
> @gongled Hi! I hope you doing well. Is there any news about this PR?

I removed `checksum.sha512` file, but I do not drop JDK version to LTS version. Unfortunately, Gradle 6.8.2 can be only build using JDK 13. Also, Gradle 6.8.x is required to build Kafka 2.7.0 (https://github.com/essentialkaos/kaos-repo/pull/253).

Do you have any ideas how can I improve this RPM spec?

**Comment by andyone** [comment]
> Unfortunately, Gradle 6.8.2 can be only build using JDK 13.

We use packages with AdoptOpenJDK binaries. Currently, [there is no JDK 12-15](https://adoptopenjdk.net/index.html).

**Comment by andyone** [comment]
> Currently, there is no JDK 12-15.

I was wrong; these versions in the archive. But I dropped them due to a lack of time for supporting and updating non-LTS versions.

**Comment by gongled** [comment]
OK. It comes across as there is no chance to get this merged. Closed.

**Pull request closed without merging**